logprep/util/validators.py

```diff
@@ -164,3 +164,13 @@ def min_len_validator(_, attribute, value, min_length):
     #  (including the tests).
     if len(value) < min_length:
         raise ValueError(f"Length of '{attribute.name}' must be => {min_length}: {len(value)}")
+
+
+def one_of_validator(_, attribute, value, member_list):
```

Comment:
I didn't find an explicit test for this validator, could you add this please?

Reply:
sure I can :-D